apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -258,20 +259,8 @@ export const moveResource = async (
 // - The immediate siblings of the resource (if any)
 export const getLocalisedSitemap = async (
   siteId: number,
-  resourceId?: number,
+  resourceId: number,
 ) => {
-  if (resourceId === undefined) {
-    // Provide a default sitemap with a single resource
-    return {
-      id: "0",
-      layout: "content",
-      title: "Root",
-      summary: "",
-      lastModified: new Date().toISOString(),
-      permalink: "/",
-    }
-  }
```

Comment:
case removed because we know ahead of time on the frontend if we need a `sitemap` and if we don't, i passed this exact same one (or a collection one).

this case only occurs for page creation, so i strengthened this argument to take `resourceId`